app/templates/index.hbs

```diff
@@ -57,3 +60,51 @@
     </div>
   </div>
 </main>
+
+<AuModal
+  @title="Geef een naam en beschrijving"
+  @modalOpen={{this.showModal}}
+  @closeModal={{this.closeModal}} as |Modal| >
+  <Modal.Body>
+    <div class="au-o-grid au-o-grid--small">
+      <div class="au-o-grid__item au-1-1">
+        <AuLabel for="input-filelabel"
+          @required={{true}}
+        >
+          Naam
+        </AuLabel>
+        <AuInput
```

Comment:
Would it make sense to make this field fill the full width in case users want to type a longer name?